src/components/Sort/Sort.js

```diff
@@ -37,7 +37,7 @@ const Sort = () => {
                 key={id}
                 className="sort-option"
                 onClick={() => {
-                  sortPage({ sort });
+                  sortPage(`${sort}`);
```

Comment:
```suggestion
                  sortPage(sort);
```
이 부분 이렇게 넣으면 위에 있는 sort를 그대로 가져와 사용할 수 있겠습니다!